chromium_src/chrome/browser/profiles/profile_manager.cc

```diff
@@ -4,4 +4,16 @@
  * You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 #include "brave/browser/profiles/brave_profile_manager.h"
+#include "brave/components/tor/tor_constants.h"
+
+// Do this for legacy tor profile migration because tor profile might be last
```

Comment:
we can handle this in a pref migration for `kProfileLastUsed`

Reply:
I tried to fix this via a migration in https://github.com/brave/brave-core/commit/ade138b18b17ee938f84750d0ab30ee583e44ed3, but I'm not sure if this is exactly what you meant (I've experimented with other ways of doing it, but this seemed to be the most reasonable to me).

@bridiver PTAL and let me know what you think, thanks!

Reply:
Actually, the right patch is https://github.com/brave/brave-core/commit/8df0db6b3f8529ce152e017ce33ea247b386d57f

Reply:
yep, that's what I was thinking